docs/diagrams/class/Time.puml

```diff
@@ -0,0 +1,55 @@
+@startuml
+
+skinparam classAttributeIconSize 0
+
+
+
+abstract class View {
```

Comment:
`protected LocalDate startOfView`
`protected final DateTimeFormatter dateFormatter`
Are these attributes missing?

Reply:
Sorry not missing, the visibility should be protected and not private. Use `#` instead of `-` to show protected variables